backend/app/Http/Controllers/Web/Operator/BatchController.php

```
@@ -244,6 +247,118 @@ public function toggleChecklistItem(Request $request, BatchStep $batchStep, Temp
+            } else {
+                $attrs += $this->scalarOutputAttrs($request, $output);
+            }
+        } catch (\InvalidArgumentException $e) {
+            return back()->with('error', __('The uploaded file is not a valid image.'));
+        } catch (\Illuminate\Validation\ValidationException $e) {
+            throw $e;
+        }
+
+        // Overwrite: soft-delete any live value, then insert the new one (keeps the
+        // partial-unique index happy and preserves the prior value as audit).
+        BatchStepOutputValue::where('batch_step_id', $batchStep->id)
```

PHPStan (2.2.7)

[error] Call to an undefined static method App\Models\BatchStepOutputValue::where().


(staticMethod.notFound)